os/kernel/init/os_start.c

```diff
@@ -828,6 +828,10 @@ void os_start(void)
 	/* Create initial tasks and bring-up the system */
 
 #ifdef CONFIG_PM
+	/* Registeer Special Domains, which are specific to Kernel*/
+	pm_domain_register("PM_IDLE_DOMAIN");
+	pm_domain_register("PM_LCD_DOMAIN");
+	pm_domain_register("PM_BLE_DOMAIN");
```

Comment:
IDLE and LCD is managing the specially for controll PM_NORMAL, PM_IDLE.
For this, we need to register during the booting, 
idle 0 , lcd 1

but ble domain shouldn't register in here

Reply:
Do we still register LCD domain even CONFIG_LCD is not enabled?
I mean, shouldn't it be:
```
#ifdef CONFIG_LCD
pm_domain_register("PM_LCD_DOMAIN");
#endif
```

Reply:
@ewoodev Done
@edwakuwaku we will consider this during the implementation of LCD Design